rasa/core/actions/action.py

```diff
@@ -501,14 +498,15 @@ async def run(
         # only utter the response if it is available
         evts = await super().run(output_channel, nlg, tracker, domain)
 
-        return evts + [UserUtteranceReverted(), UserUtteranceReverted()]
+        return [*evts, UserUtteranceReverted(), UserUtteranceReverted()]
```

Comment:
Out of curiosity, does this change come from ruff, or is it unrelated?

Reply:
🤔 that's a mistake, I have disabled the check [here](https://github.com/RasaHQ/rasa/pull/12138/files#r1137088630). The change comes from running ruff with `--fix`. Since you commented on this, what do you think of this? at first sight using `*` seems a bit overkill to me, and I couldn't find documentation on why this check was implemented in ruff

Reply:
I expect this to be optimisation, so it would be good to see it as a hint but not as a failure.